client/export.go

```diff
@@ -36,10 +37,23 @@ func (c *Client) Export(key, singleKeyExportFormat string) (string, error) {
 		sortedKeys = append(sortedKeys, secret.Key)
 		keySecretMap[secret.Key] = secret
 	}
+
+	var AppendMap map[string]string
+	AppendMap = make(map[string]string)
+	for _, val := range c.Append {
+		parts := strings.Split(val, ":")
+		if len(parts) == 2 {
+			AppendMap[parts[0]] = parts[1]
+		} else {
+			return "", errors.New("Bad append format (--append <variable>:<string>)")
```

Comment:
On the client level, I expect to get a proper data structure like `map[string]string`, not a string that we need to parse. Let's parse it on cmd package.
Be aware that the treasury Client package can be imported from other tools/packages e.g.: from Lambda Go Function.